web-api/src/business/useCases/trialSessions/setTrialSessionCalendarInteractor.ts

```diff
@@ -144,10 +145,8 @@ export const setTrialSessionCalendarInteractor = async (
     // Story: 10422
```

Comment:
If this comment proved useful, it may be worth updating it to include some information about why we're using the `p-queue` module here as opposed to the previous approach.